#! /bin/sh /usr/share/dpatch/dpatch-run
## mpegrec_no_backward_compat.dpatch by Daniel Kristjansson <danielk@cuymedia.net>

@DPATCH@
diff -urNad mythtv-0.20+fixes15192~/libs/libmythtv/mpegrecorder.cpp mythtv-0.20+fixes15192/libs/libmythtv/mpegrecorder.cpp
--- blah/libs/libmythtv/mpegrecorder.cpp	(revision 15199)
+++ blah/libs/libmythtv/mpegrecorder.cpp	(working copy)
@@ -350,6 +350,9 @@
 
 bool MpegRecorder::SetIVTVDeviceOptions(int chanfd)
 {
+    (void) chanfd;
+    return false; // we don't need backward compatibility in LMCE.
+
     struct ivtv_ioctl_codec ivtvcodec;
     bzero(&ivtvcodec, sizeof(ivtvcodec));
 
